packages/onegraph-apollo-client/src/OneGraphApolloClient.js

```diff
@@ -21,13 +23,48 @@ class OneGraphApolloClient extends ApolloClient {
     validateConfig(config);
     const {oneGraphAuth} = config;
     const uri = new URL(oneGraphAuth.oneGraphOrigin);
-    uri.pathname = '/dynamic';
+    uri.pathname = '/graphql';
     uri.searchParams.append('app_id', oneGraphAuth.appId);
 
+    const httpLink = new HttpLink({
+      uri: uri,
+    });
+
+    const subscriptionClient = new SubscriptionClient(oneGraphAuth.appId, {
+      oneGraphAuth: oneGraphAuth,
+      host: uri.host,
```

Comment:
Set `lazy: true` to delay the ws creation until they start a subscription. This way people we don't create unnecessary connections.

```suggestion
      host: uri.host,
      lazy: true,
```